sinker/otel/bridgeservice/bridge.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"sort"
 	"time"
+  "strings"
```

Comment:
not used?

Reply:
that was a conflict resolution, resolved.